packages/nextclade/src/qc/ruleSnpClusters.cpp

```diff
@@ -72,8 +87,13 @@ namespace Nextclade {
       return {};
     }
 
+    // NOTE: we exclude reversions of deletions
+    // See: https://github.com/nextstrain/nextclade/issues/707
+    const auto privateSubstitutions = eraseIntersection(query.privateNucMutations.privateSubstitutions,
+      query.privateNucMutations.reversionsOfDeletions);
+
     // TODO: should we also account for result. privateDeletions here
-    const auto snpClusters = findSnpClusters(query.privateNucMutations.privateSubstitutions, config);
+    const auto snpClusters = findSnpClusters(privateSubstitutions, config);
```

Comment:
Because the changes touch only the `ruleSnpCluster.cpp` file, this should have no impact on anything other than that QC rule within Nextclade, correct?

Reply:
Right